src/symbolic.jl

```diff
@@ -86,14 +86,14 @@ simplify(n::Number) = n
 simplify(s::SymbolicVariable) = s
 
 # The default is just to simplify arguments.
-simplify{T}(x::SymbolParameter{T}, args) = Expr(:call, T, map(simplify, args)...)
+simplify(x::SymbolParameter{T}, args) where {T} = Expr(:call, T, map(simplify, args)...)
 
 function simplify(ex::Expr)
     if ex.head != :call
         return ex
     end
     if all(isnumber, ex.args[2:end]) && length(ex.args) > 1
-        return eval(current_module(), ex)
```

Comment:
Note that this is wrong.

Reply:
Ok. Do you know where it is documented how to upgrade code like this?

Reply:
I mentioned it in https://github.com/johnmyleswhite/Calculus.jl/pull/118. There will be no dynamically scoped current module so you need to thread it through manually.